pontoon/sync/tests/formats/test_ftl.py

```diff
@@ -0,0 +1,291 @@
+import os
+import shutil
+import tempfile
+
+from textwrap import dedent
+
+from django_nose.tools import (
+    assert_equal,
+    assert_raises,
+    assert_true,
+)
+from pontoon.base.tests import (
+    assert_attributes_equal,
+    create_named_tempfile,
+    LocaleFactory,
+    TestCase,
+)
+from pontoon.sync.formats import ftl
+from pontoon.sync.tests.formats import FormatTestsMixin
+
+
+class FTLResourceTests(FormatTestsMixin, TestCase):
+    def setUp(self):
+        super(FTLResourceTests, self).setUp()
+        self.tempdir = tempfile.mkdtemp()
+
+    def tearDown(self):
+        super(FTLResourceTests, self).tearDown()
+        shutil.rmtree(self.tempdir)
+
+    def get_nonexistant_file_resource(self, path):
+        contents = dedent("""text = Arise, awake and do not stop until the goal is reached.""")
```

Comment:
No need to use `"""` here, simple quotes will suffice. And `dedent` is likewise useless.